transplant_guide.md

```diff
@@ -120,29 +124,38 @@ Known as the most LGBT friendly neighborhood. Tons of cool shops and restaurants
+
+**Beacon Hill**
+Some places in this neighborhood will be close to the light rail station, which makes it an easy commute into downtown (and also to the airport). It is further south than most of the neighborhoods listed here, but especially if you are close to the light rail, a house here can be cheaper and a good find. There are a couple little retail areas around the light rail with a grocery store, coffee shops, a library and a few bars and restaurants.
 
 **Central District**  
-Info about CD
+This neighborhood is further east than capitol hill/first hill/the ID, but there is more residential and sometimes cheaper housing available. It is bikeable to downtown (hills not withstanding), and is generally quieter evenings/weekends than neighborhoods with more of a night life. There are a few cute neighborhood bars and good coffee shops though, like Chucks Hop Shop CD, Neighbor Lady, Twilight Exit, Cortona Cafe etc.  
 
 **First Hill**  
-Info about First Hill
+Used to also be referred to as pill hill, because there are a lot of hospitals in the area. If you are easily awoken by ambulance sirens, it may not be the best place to live. The location is very convenient as it is close to downtown and to Capitol Hill, but there is not often a ton of open housing in the area. There are a few new complexes under construction in the area but will likely be pretty pricey when they open, and there is at least one microhousing building in the neighborhood.
 
 **Fremont**  
-Info about Fremont
+Fremont is the self-coined Center of the Universe, and can be a fun place to live if you can find housing-- it often is sparse and expensive. It is the location of the solstice parade (which includes a naked bike ride) and is home to many bars, restaurants, breweries and tap rooms. It is north of queen anne from downtown, or west of Wallingford on the north side of lake union. From downtown, there is a bridge that is low enough it has to open for boat traffic, so that can cause some traffic problems from time to time. It is on the Burke Gilman trail (fun bike ride) and it not too terrible of a bike commute to downtown(but again, hills, because Seattle).
 
 **Ballard**  
-Info about Ballard
+Ballard is kind of new hipster central in Seattle, with new craft cocktail gastropubs and microbreweries seemingly popping up weekly. It is a pretty fun place to go out on the weekends and they have an awesome Sunday farmer's market. There is quite a bit of new construction, but it is pretty pricy given it is brand new.
+Getting downtown via transit isn't impossible, but because it is further west, it can make getting to sort of inconvenient at times.  It would not be convenient to commute to an east side (aka Bellevue (Expedia/Concur) or Redmond (Microsoft)) internship or job, but if you are coming from a place where you have an hour+ commute, maybe it wouldn't be that noticeable to you.  
```

Comment:
I disagree with this last paragraph. There are several bus routes that will get you over to the 542/545 within 10-20 minutes, which will not add much more to an eastside commute than living in the u district. There are also several great routes going downtown, including the rapidride D, and that's 10-30 minutes depending on where in Ballard.